packages/shared/src/components/fields/TextField.module.css

```diff
@@ -10,7 +10,6 @@
 
   &:global(.invalid) {
     box-shadow: inset 0.125rem 0 0 0 var(--theme-status-error);
-    border-color: var(--theme-status-error);
```

Comment:
Why are we dropping this?

Reply:
According to the guideline this is a design for a text field with Error status: 

![Screenshot 2022-01-19 at 13 26 56](https://user-images.githubusercontent.com/33021204/150129461-c4b1a399-991f-47ae-b617-78a995ae5568.png)

With `border-color: var(--theme-status-error);` the full border applies:

![Screenshot 2022-01-19 at 13 41 40](https://user-images.githubusercontent.com/33021204/150132886-a4dca827-4f99-43fa-8523-ff6e99c7020e.png)

Please let me know if misinterpreted something.

Reply:
Ok, you're right! Let's proceed :)